docs/getting-started.txt

```diff
@@ -1,300 +1,311 @@
-#################
- Getting Started
-#################
+#################################
+ Quickstart for Model Developers
+#################################
 
-This section walks through the steps to run examples in Determined from zero. You should be able to:
+This quickstart walks you through the basic steps to install and run Determined, run training
+experiments, and visualize training experiment results in your browser.
 
-#. Get a light touch with Determined concepts and features;
-#. Install Determined;
-#. Get familiar with the WebUI and the CLI;
-#. Train an example model with a single GPU;
-#. Train an example model with distributed training;
-#. Train an example model with hyperparameter tuning.
+Three model data and code examples use the MNIST dataset to demonstrate:
 
-*********************
- System Requirements
-*********************
+-  training on a local, single CPU with or without a GPU.
+-  how to run a distributed training job on multiple GPUs to improve performance.
+-  how to use hyperparameter tuning.
 
-Please review :doc:`this document </sysadmin-deploy-on-prem/requirements>` to see if your system
-meets the requirements to run Determined, including instructions on how to install the Docker.
+A Determined cluster comprises a master and one or more agents. A master provides centralized
+resource management of agents installed on distributed resources. Master and agent behavior are
+controlled by setting configuration variables. Each of these examples uses the same model code and
+data with simple configuration changes to account for different performance requirement and resource
+utilization.
```

Comment:
typo: performance requirement -> performance requirements
suggestion: I wouldn't characterize the configuration changes as being due to different "performance requirement(s)".  For instance, hyperparameter search isn't a "performance requirement" -- it's an experiment, usually exploratory.

Reply:
Fixed. Wasn't a very good sentence anyway.